Topic 5: NoSQL - MongoDB

hherminioandres opened the pull request:
1- Using the same SQL-MySql data model, map it to the corresponding database, collections and documents.
2- Fetch all students whose notes in a specific course were greater than 4.
3- Fetch all courses ordered by name for a given teacher.
4- Create a new project using Maven and Morphia to map the objects created in 1. Create the necessary DAOs to get same result than 2

hherminioandres committed:
Topic 5: NoSQL - MongoDB

1- Using the same SQL-MySql data model, map it to the corresponding database, collections and documents.
2- Fetch all students whose notes in a specific course were greater than 4.
3- Fetch all courses ordered by name for a given teacher.
4- Create a new project using Maven and Morphia to map the objects created in 1. Create the necessary DAOs to get same result than 2

sebastian-garofalo commented on topic_5/.gitignore:
```diff
@@ -0,0 +1,11 @@
+topic_5/.project
+topic_5/.classpath
+topic_5/target/classes/com/globant/dao/MongoConnection.class
```
You can ignore a directory, in this case /target instead of ignoring files one by one

sebastian-garofalo commented on topic_5/src/main/java/com/globant/dao/MongoConnection.java:
```diff
@@ -0,0 +1,70 @@
+import com.mongodb.ServerAddress;
+
+/**
+ * 
```
remember to add a simple description of the class

sebastian-garofalo commented on topic_5/src/main/java/com/globant/dao/MongoConnection.java:
```diff
@@ -0,0 +1,70 @@
+	}
+
+	public static void main(String[] args) {
+		MongoConnection mongo = new MongoConnection();
```
Please format the code before submiting

sebastian-garofalo commented on topic_5/src/main/java/com/globant/dao/MongoConnection.java:
```diff
@@ -0,0 +1,70 @@
+	}
+	
+	public void configuration() {
+	     MongoClient mongo = new MongoClient(new ServerAddress("127.0.0.1", 27017));
```
Connection info, DB names etc could be extracted as constants

hherminioandres committed:
Topic 5: Several corrections

- Applied format
- Create the "Schedule" class
- Comments added in the classes
- Modified gitignore file
- Typo corrections
